src/authentication/authentication.config.ts

```diff
@@ -48,7 +48,7 @@ function authenticationConfig ($authProvider: AuthProvider, Constants) {
       $authProvider.oauth2(_.merge(customConfig, {
         url: Constants.baseURL + 'auth/oauth2',
         oauthType: '2.0',
-        redirectUri: window.location.origin,
+        redirectUri: window.location.origin + window.location.pathname,
```

Comment:
Why not removing trailing slash to handle http://domain.org/ properly and avoid impact for current users ?

Reply:
I do think it is cleaner to always include the trailing slash, but I see you point. Updating the PR.